Vagrantfile

```diff
@@ -5,6 +5,8 @@
 LINUX_BASE_BOX = "bento/ubuntu-16.04"
 FREEBSD_BASE_BOX = "freebsd/FreeBSD-11.2-STABLE"
 
+LINUX_IP_ADDRESS = "10.199.0.200"
```

Comment:
Does this mean we could use http://10.199.0.200:4646 in documentation? I didn't know that was possible!

Will this conflict with with any IPs already on the machine or does it create a local-only interface?

Reply:
> Does this mean we could use http://10.199.0.200:4646 in documentation? I didn't know that was possible!

Yes, in local dev, but needs to bind nomad to 0.0.0.0 instead of the default 127.0.0.1 .

> Will this conflict with with any IPs already on the machine or does it create a local-only interface?

Conflicts will mess up routing indeed, but not sure how to protect against it.  10.199.0.0/16 isn't a commonly used block and we already use it elsewhere in the Vagrant file.